src/mtcp/mtcp_sys.h

```diff
@@ -483,6 +484,13 @@ static unsigned int myinfo_gs;
   INLINE_SYSCALL_RAW(__ARM_NR_set_tls, 1, args)
 # endif // if defined(__arm__)
 
+# ifdef __arm__
+// https://elixir.bootlin.com/linux/v3.1/source/arch/arm/include/asm/unistd.h#L411
+#  define __ARM_NR_cacheflush 0x0f0002
+#  define mtcp_sys_kernel_cacheflush(args ...) \
+  INLINE_SYSCALL_RAW(__ARM_NR_cacheflush, 2, args)
```

Comment:
Could you check if the number of argument is correct? (see [man cacheflush](http://man7.org/linux/man-pages/man2/cacheflush.2.html)